charts/kube-prometheus-stack/templates/prometheus/rules-1.14/alertmanager.rules.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     - alert: AlertmanagerFailedReload
       annotations:
         description: Configuration has failed to load for {{`{{`}} $labels.namespace {{`}}`}}/{{`{{`}} $labels.pod{{`}}`}}.
-        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-alertmanagerfailedreload
+        runbook_url: https://runbooks.prometheus-operator.dev/runbooks/alertmanager/alertmanagerfailedreload
```

Comment:
Not sure if this change is intended as it removes the runbookURL functionality.

Reply:
Thanks for pointing me to this

It was decided [here](https://github.com/prometheus-operator/kube-prometheus/pull/1267) to switch to this new site  https://runbooks.prometheus-operator.dev/ based on this [new repo](https://github.com/prometheus-operator/runbooks/tree/main/content/runbooks) 

Problem is that it contains different spaces for different rules, for example,
 
https://runbooks.prometheus-operator.dev/runbooks/alertmanager/
https://runbooks.prometheus-operator.dev/runbooks/prometheus-operator/
https://runbooks.prometheus-operator.dev/runbooks/general/
https://runbooks.prometheus-operator.dev/runbooks/kubernetes/
https://runbooks.prometheus-operator.dev/runbooks/kube-state-metrics/
https://runbooks.prometheus-operator.dev/runbooks/node/
https://runbooks.prometheus-operator.dev/runbooks/prometheus/

Do you think we should replace all of them to continue supporting `.Values.defaultRules.runbookUrl`
or keep the suggested source for runbooks and remove the `runbookUrl` functionality?

@bismarck @gianrubio @gkarthiks @scottrigby @vsliouniaev @Xtigyro @mrueg